### DISCUSSION TITLE
What will happen with user that use one-click login flow if they enable MFA

### DISCUSSION THREAD

**Discussion by vonhathanh** [Web3Auth/Web3Auth#1020] [Help]
<img width="785" alt="Screen Shot 2022-12-07 at 10 05 21" src="https://user-images.githubusercontent.com/20921387/206081694-de9969af-a1ac-46b2-a8ef-3503fcc1dec3.png">
According to the picture above, users that don't enable MFA will use the key share as their private key. 
My question is what happen when they enable MFA and add the other two shares, because if they do that, the private key will be changed and the user's address will be changed too.

**Comment by shahbaz17** [comment]
Hey @vonhathanh 

Please check out https://main.d9npg8ay86zyo.amplifyapp.com live demo and try to run through the scenario. In both cases, it should return the same address.
